crates/rolldown/tests/rolldown/topics/live_bindings/sourcemap_cjs/artifacts.snap

```diff
@@ -0,0 +1,74 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+snapshot_kind: text
+---
+# Assets
+
+## main1.js
+
+```js
+"use strict";
+const require_shared = require('./shared.js');
+
+//#region main1.js
+function main() {
+	require_shared.shared();
+}
+main();
+
+//#endregion
+//# sourceMappingURL=main1.js.map
+```
+## main2.js
+
+```js
+"use strict";
+const require_shared = require('./shared.js');
+
+exports.shared = require_shared.shared
+```
+## shared.js
+
+```js
+"use strict";
+
+//#region shared.js
+function shared() {
+	console.trace("shared");
+}
+
+//#endregion
+Object.defineProperty(exports, 'shared', {
+  enumerable: true,
+  get: function () {
+    return shared;
+  }
+});
+//# sourceMappingURL=shared.js.map
+```
+
+# Sourcemap Visualizer
+
+```
+- ../main1.js
+(2:0-2:9) "\nfunction" --> (4:0-4:9) "\nfunction"
+(2:9-2:16) " main()" --> (4:9-4:16) " main()"
+(2:16-3:2) " {\n " --> (4:16-5:0) " {"
+(3:2-0:0) "" --> (5:0-5:1) "\n"
+(0:0-3:2) "import {shared} from './shared.js'\n\nfunction main() {\n " --> (5:1-5:16) "\trequire_shared"
```

Comment:
This looks wrong, but not sure what's causing this.

Reply:
Fee free to add cases in https://github.com/rolldown/rolldown/issues/2737.

I'll notice this too. But it's too cumbersome to solve them now. The issue chain is too long.

Reply:
Here caused by `SourcemapVisualizer`, the `\trequire_shared` corresponding token should remapping to `import {shared} from './shared.js'\n`. Here i meet some issues with read content cross lines, so here print has some misleading.

Reply:
@underfin  So there's also bug of `SourcemapVisualizer`? I don't think we need to fix it now, but would be good to fix it now. I'll record this in https://github.com/rolldown/rolldown/issues/2737.

Reply:
Yeah. Let us track to it.